tests/common.py

```diff
@@ -0,0 +1,8 @@
+class Response:  # For testing 'log_handler'
```

Comment:
I have two thoughts here.

1. You may want to be more explicit that this isn't the actual `Response` object that is used by our birds. An easy way to do this would be to call it `TestResponse`. That way it is clear that this is just for testing. If, for some reason, we can't change the name, put a docstring here explaining that it is just a dummy.

2. Have you tried using the actual `Response` object? If you are able to I think that would be preferrable.

Reply:
I was hoping to use the actual `Response` object, but it doesn't have the `update_status` function when called from the test file since it's a `WpsTestResponse` rather than a `WPSResponse`. Renaming this object to `TestResponse` should work.

Reply:
Sounds good!